### PR TITLE
feat: add lipophilicity dataset

### PR DESCRIPTION
**Pull request by MicPie**
See issue https://github.com/OpenBioML/chemnlp/issues/22.

### CODE REVIEW

**Comment by kjappelbaum** [comment]
>@kjappelbaum Do you know how I can write the bibtex string to the meta.yaml so it does pass the check, because with the current setup the notebook here and in the freesolv dir will produce a meta.yaml that will not pass the checks.

i think freesolv passed the check? I'll give it a look in a sec

**Comment by MicPie** [comment]
yes, freesolv passed, but it wouldn't when you would recreate the meta.yaml based on the notebook.

File: data/lipophilicity/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  type: continuous
+  names:
+  - octanol/water distribution coefficient (logD at pH 7.4)
+  - octanol/water distribution coefficient
+identifiers:
+- id: SMILES
+  type: SMILES
+  description: SMILES
+license: CC BY-SA 3.0
+links:
```
Comment (kjappelbaum) [inline data/lipophilicity/meta.yaml:18]: Shall we leave them like this? 

Originally I thought of adding some more context with "description" etc. 
I'm ok with dropping this (but there should be one link we highlight as the one where the data comes from). 


I also realized that this part is currently not validated.

Comment (MicPie) [reply data/lipophilicity/meta.yaml:18]: Do you mean the description at the top or for the license or links?

Comment (kjappelbaum) [reply data/lipophilicity/meta.yaml:18]: what I mean is to replace the `links` with something like 

```
links:
  - name: dataset
    url: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/OVHAW8
    description: Original dataset
```

then we could at least identify what link refers to the original dataset and which are other references.

Comment (MicPie) [reply data/lipophilicity/meta.yaml:18]: ok, got it, I added a url description but no name field.

File: data/lipophilicity/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- url: https://deepchem.readthedocs.io/en/latest/api_reference/moleculenet.html#lipo-datasets
+  description: original dataset link from moleculenet
+- url: https://www.ebi.ac.uk/chembl/document_report_card/CHEMBL3301361/
+  description: original report card
+- url: https://chembl.gitbook.io/chembl-interface-documentation/about#data-licensing
+  description: original dataset license from chembl
+- url: https://creativecommons.org/licenses/by-sa/3.0/
+  description: used dataset license
+num_points: 4200
+url: https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/Lipophilicity.csv
```
Comment (kjappelbaum) [inline data/lipophilicity/meta.yaml:32]:
```suggestion
url: https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/Lipophilicity.csv
```

Comment (kjappelbaum) [reply data/lipophilicity/meta.yaml:32]: or do you think it is better to keep it here. I'm fine with both as long as we have a way of easily finding the original dataset

Comment (MicPie) [reply data/lipophilicity/meta.yaml:32]: fixed

Comment (MicPie) [reply data/lipophilicity/meta.yaml:32]: also the reason for the strange yaml multiline behavior here was the special characters, after the removal it worked.

**Pull request merged**